package.json

```diff
@@ -61,11 +61,13 @@
     "css-loader": "^3.2.0",
     "dotenv": "^8.1.0",
     "eslint": "5.16.0",
+    "eslint-config-prettier": "^6.4.0",
```

Comment:
eslint was not working properly without these packages.

Reply:
Ah. That would explain things. I think I have this installed globally.